front-end-poc/src/ModelLookup/ModelDiagram.tsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
+import ReactDOM from 'react-dom'
 import { IModelVersion } from "./types"
+import {example1, unmarshall, LinearRegressionHandler} from "@manstis/pmml-linear-model-viewer"
+import * as PMML from "@manstis/pmml-linear-model-viewer/src/generated/www.dmg.org/PMML-4_4"
```

Comment:
I don't like this at all; however I _should_ be able to export `* as PMML` in my module [one day](https://github.com/tc39/ecma262/pull/1174).

Reply:
@kelvah Is there a better way to re-export an imported `* as ns from "..."`?

Reply:
@manstis In your module you can first `import * as PMML from "..."` and then export PMML along the other exports. The result should be the same as `export * as PMML from "..."`